internal/services/consumer.go

```diff
@@ -57,6 +57,12 @@ var (
 		Namespace: "strimzi_canary",
 		Help:      "The total number of consumers not joining the group within the timeout",
 	}, []string{"clientid"})
+
+	refreshConsumerMetadataError = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
We might also need to update the Metrics section in README file

Reply:
Good point

Reply:
Done